vite.config.ts

```diff
@@ -1,7 +1,7 @@
-import { defineConfig } from 'vite'
-import react from '@vitejs/plugin-react'
-
+import { defineConfig } from "vite";
+import react from "@vitejs/plugin-react";
+import relay from "vite-plugin-relay";
 // https://vitejs.dev/config/
 export default defineConfig({
-  plugins: [react()]
-})
+  plugins: [react(), relay],
```

Comment:
Vite doesn't allow us to customize babel plugin, so here use vite-plugin-relay for relay